packages/cli/src/lib/commands/init/processes/generate-lh-budgets.ts

```diff
@@ -1,54 +1,71 @@
+import { join } from 'node:path';
+import { existsSync, mkdirSync } from 'node:fs';
+import { cwd } from 'node:process';
+
 import { RcJson } from '../../../types';
-import { join } from 'path';
 import { readFile, writeFile } from '../../../core/file';
-import { log, logVerbose } from '../../../core/loggin';
-import { existsSync, mkdirSync } from 'fs';
+import { logVerbose } from '../../../core/loggin';
 import { BudgetsExampleMap } from '../constants';
 import { ifThenElse } from '../../../core/processing/behaviors';
 import { CLIProcess } from '../../../core/processing/types';
 import { deriveBudgetsFromLhr } from '../derive-budgets-from-lhr';
 
-export async function generateLgBudgets(cliCfg: RcJson & { lhr?: string }): Promise<RcJson> {
-  const destPath = process.cwd();
-  const targetFileName = 'budget.json';
-  let fileContent = '';
+export const budgetsFileExist = (path: string) => {
+  return readFile(path) !== '';
+};
+
+export const defaultBudgets = () => {
+  logVerbose('New budgets used');
+  const tplFileName = BudgetsExampleMap['budgets'];
+  const exampleSourceLocation = join(__dirname, '..', 'static', tplFileName);
+  return readFile(exampleSourceLocation, { fail: true }).toString();
```

Comment:
OOS note: If possible, I suggest distinguishing `readFile` which, if you call `readFile` from Node and include `utf8`, automatically types it as a `string`, and `readJsonFile` for JSON, so that the `toString` is not needed.

Additionally, the type generics seems to be forcing you to use `as never` when mocking the return value in tests.

Reply:
Agreed, we should refactor the helpers. Also readFile does not do the same as readFile does from node